spec.md

```diff
@@ -2450,6 +2450,26 @@ asClock(x)
 
 [@sec:primitive-operations] will describe the format and semantics of each primitive operation.
 
+## Primitive Property Operations {#sec:expressions:primitive-property-operations}
+
+All fundamental operations on property types are expressed as a FIRRTL primitive operation.
+In general, each operation takes some number of property type expressions as arguments.
+
+The general form of a primitive property operation is expressed as follows:
+
+```firrtl
+op(arg0, arg1, ..., argn)
+```
+
+The following examples of primitive property operations demonstrate adding two property expressions, `a`{.firrtl} and `b`{.firrtl}, and adding an integer property literal to expression `a`{.firrtl}.
```

Comment:
Nit: adding the `{.firrtl}` hint to the syntax highlighter doesn't change the format. It's usually cleaner to drop it. This is _not_ the case for `Integer` later, though. That does add useful highlighting.